base/cvd/cuttlefish/host/libs/web/credential_source.h

```diff
@@ -29,6 +30,13 @@ class CredentialSource {
   virtual Result<std::string> Credential() = 0;
 };
 
+enum class CredentialSourceId {
+    kAcloud,
```

Comment:
nit: Acloud is not a good name for this. These credentials are for the Google-managed Android build servers available on ci.android.com. Maybe `kAndroidCI` or `kGoogleAndroid`?

Reply:
will do.